api/tests/Behat/bootstrap/v2/Common/AssertTrait.php

```diff
@@ -43,6 +43,29 @@ public function assertIntEqualsInt(
         );
     }
 
+    public function assertBoolIsTrue(
+        $expected,
+        string $comparisonSubject
+    ) {
+        assert(
+            true == $expected,
+            $this->getAssertMessage($expected, 'false', $comparisonSubject)
+        );
+    }
+
+    public function assertStringDoesNotContainString(
+        $notExpected,
+        $found,
+        string $comparisonSubject
+    ) {
+        $foundFormatted = strval(trim(strtolower($found)));
+        $notExpectedFormatted = strval(trim(strtolower($notExpected)));
+        assert(
+            !str_contains($foundFormatted, $notExpectedFormatted),
```

Comment:
I think this is a PHP 8 only function, is it working? I do remember seeing some Symfony polyfill code for this a while ago so maybe that's helping us out here? https://www.php.net/manual/en/function.str-contains.php

Reply:
Doesn't complain and I've used it a few times before. Didn't realise it's php8!

Reply:
If you want me to use a replacement I can but have left in as it's doing it's thang (and we're going to upgrade to php8 soonish anyway)

Reply:
Naw its all good if its working. Like you said we'll be there soon so makes sense to use the shiny things when we can.